app/iht/utils/tnrb/TnrbHelper.scala

```diff
@@ -38,6 +38,13 @@ object TnrbHelper {
   val deceasedWidowCheckDatePage= iht.controllers.application.tnrb.routes.DeceasedWidowCheckDateController.onPageLoad()
   val deceasedWidowCheckQuestionPage = iht.controllers.application.tnrb.routes.DeceasedWidowCheckQuestionController.onPageLoad()
 
+  def addFragmentIdentifier(call:Call, identifier:Option[String] = None) = {
```

Comment:
Don't duplicate this code - it is also in estate controller. Put it in one place and then call it there.